tools/benchmark/src/ResultTypes.ts

```diff
@@ -64,7 +64,7 @@ export interface BenchmarkData {
 	/**
 	 * Custom data formatters.
```

Comment:
We should document the expected format of this. What are the keys? What are the values?

Reply:
`customDataFormatters` removed so no longer applies 😄 .